client/homescreen/style.scss

```diff
@@ -70,3 +70,23 @@
 	height: 225px;
 	background-color: #6495ed;
 }
+
+
+.is-inbox {
```

Comment:
Minor nitpick: not sure if we'll keep this with the above comment, but since this is a state class, it would be good to combine it with the selector its being applied to.  I.e., `.woocommerce-homescreen-column.is-inbox`